Source/Data/SpectrumAnalysis/KTSeqLine.hh

```diff
@@ -0,0 +1,129 @@
+/*
+ * KTSeqLine.hh
+ *
+ *  Created on: Sep 8, 2016
+ *      Author: Christine
+ */
+
+#ifndef KTSEQLINE_HH_
+#define KTSEQLINE_HH_
+
+
+#include "KTFrequencyDomainArray.hh"
```

Comment:
Why are KTFrequencyDomainArray.hh and KTPhysicalArray.hh included?  I might just be missing where they're used, but I don't see it.